src/python/tools/configuration/marius_config.py

```diff
@@ -873,16 +883,28 @@ def load_config(input_config_path, save=False):
 
     if output_config.storage.model_dir[-1] != "/":
         output_config.storage.model_dir += "/"
+    
+    if output_config.training.resume_from_checkpoint != "" and output_config.training.resume_from_checkpoint[-1] != "/":
```

Comment:
Should we remove the training.resume_from_checkpoint and just have users specify the checkpoint by the model_dir? Not sure how needed it is.

Reply:
I guess it is useful if you want to keep the original checkpoint intact rather than overwriting it. Your call on whether we keep it.

Reply:
yep, would be kind of useful to keep around the old checkpoint files.